cyhy/core/geoloc.py

```diff
@@ -39,3 +39,15 @@ def lookup(self, ip):
             return (response.location.longitude, response.location.latitude)
         except AddressNotFoundError:
             return (None, None)
+
+    def check_restricted_cidr(self, cidr):
+        restricted_countries = ["China","Iran","North Korea","Russia"]
+        has_restricted = False
+        try:
+            response = self.__reader.city(str(cidr[0]))
+            if response.country.name in restricted_countries:
+                print("Warning! %s traced to restricted country: %s" % (cidr, response.country.name))
+                has_restricted = True
+        except AddressNotFoundError:
+            print('Address not found.')
```

Comment:
This error message is not very descriptive.  How about
```suggestion
            print("CIDR %s not found in geolocation database", cidr)
```
instead?

Reply:
Yes, more descriptive would be better. Will note.